frontend-instructor/src/components/coursesCreate/CreateMilestone.jsx

```
@@ -0,0 +1,191 @@
+      {/* Form Modal */}
+      {showForm && (
+        <div className='fixed inset-0 bg-black/75 flex items-center justify-center z-50'>
+          <div className='bg-white rounded-lg shadow-xl max-w-md w-full mx-4 overflow-hidden'>
+            <div className='flex justify-between items-center px-6 py-4 bg-gray-100'>
+              <h2 className='text-xl font-semibold'>
+                {editingMilestone ? 'Edit Milestone' : 'Add New Milestone'}
+              </h2>
+              <button
+                onClick={handleFormClose}
+                className='text-gray-500 hover:text-gray-700 focus:outline-none'
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)